lntest/itest/lnd_test.go

```diff
@@ -1053,14 +1053,18 @@ func basicChannelFundingTest(t *harnessTest, net *lntest.NetworkHarness,
 		return nil, nil, nil, fmt.Errorf("unable to get bobs's "+
 			"balance: %v", err)
 	}
-	if aliceBal.Balance != int64(chanAmt-pushAmt-calcStaticFee(0)) {
+
+	aliceBalance := btcutil.Amount(aliceBal.Balance)
+	if aliceBalance != chanAmt-pushAmt-calcStaticFee(0) {
```

Comment:
store `chanAmt-pushAmt-calcStaticFee(0)` in var?

Reply:
done